recipes/libhdfs3/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "2.2.31" %}
+
+package:
+  name: libhdfs3
+  version: "{{ version }}"
+
+source:
+    git_url: https://github.com/PivotalRD/libhdfs3.git
+    git_tag: v{{ version }}
+
+build:
+  number: 0
+  # This package is a dependency of hdfs3, which is primarily used with HDFS on Linux.
+  skip: True  # [not linux]
+
+requirements:
+    build:
+        - protobuf
+        - libgsasl
+        - libntlm
+        - libxml2
+        - krb5
+        - libuuid
+        - boost ==1.60
```

Comment:
@koverholt exact pins like this are fragile.  They can lead to unsatisfiable dependency issues quickly.  It is much better to prefer >= (perhaps also with an upper bound) or <= specs (perhaps with a lower bound), or if those won't work, the * spec that @ocefpaf suggests.